Doc/library/multiprocessing.shared_memory.rst

```diff
@@ -64,6 +64,9 @@ copying of data.
    memory block may be larger or equal to the size requested.  When attaching
    to an existing shared memory block, the ``size`` parameter is ignored.
 
+   .. note::
```

Comment:
I think it would be better to add a sentence to the above paragraph instead of adding a note.

Reply:
done.